packages/evolution-frontend/src/actions/Survey.ts

```diff
@@ -384,3 +387,463 @@ export const startRemoveGroupedObjects = function (
         }
     };
 };
+
+export const startUpdateSurveyValidateInterview = function (
```

Comment:
These redux actions should not be in the same file as the participant app actions, as they expose admin functionalities. There can be a SurveyAdmin.ts action file for those, make sure all the copied actions are only for admins, there might be some for the participant app as well (like startSetInterview or startCreateInterview). See which components call them.

Reply:
Moved to SurveyAdmin.ts